src/protobuf/api/instance.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package api;
+
+service Instance {
+  rpc Create (CreateInstanceRequest) returns (CreateInstanceResponse) {}
+}
+
+message CreateInstanceRequest {
+  string id = 1;               // Service's sid or hash.
+  repeated string env = 2;     // Env vars to apply to service's instance on runtime.
+}
+
+message CreateInstanceResponse {
+  string sid = 1;         // Service's id.
```

Comment:
there is no reason to have the sid here. Sid is for the service, not the instance

Reply:
done